webdav.go

```diff
@@ -90,12 +91,18 @@ func (wd WebDAV) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhtt
 		FileSystem: webdav.Dir(root),
 		LockSystem: wd.lockSystem,
 		Logger: func(req *http.Request, err error) {
-			if err != nil {
-				wd.logger.Error("internal handler error",
-					zap.Error(err),
-					zap.Object("request", caddyhttp.LoggableHTTPRequest{Request: req}),
-				)
+			if err == nil {
+				return
 			}
+			// ignore errors about non-exsiting files
+			if os.IsNotExist(err) {
```

Comment:
This is great; but I think the new recommended practice is: `errors.Is(err, fs.ErrNotExist)`

Not a blocker though, I can merge this either way IMO.